.github/workflows/main.yml

```diff
@@ -111,6 +111,39 @@ jobs:
         user: __token__
         password: ${{ secrets.PYPI_TOKEN }}
 
+  publish-rsconnect:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.8.x
+    - name: Install uv # see scripts/temporary-rename
+      uses: astral-sh/setup-uv@v5
+    - run: pip install -e '.[test]'
+    - run: pip freeze
+    - run: make dist
+      id: create_dist
+      env:
+        PACKAGE_NAME: rsconnect
```

Comment:
Oh, I see. You are using a separate job to release the 'rsconnect' named version.

Is it possible to parameterize this and use the same release flow for each?

Reply:
I was worried about concurrent changes to the file resulting in publishing one but not the other but I think I just forgot how matrixed jobs work; I'll update with a matrixed approach.